src/Domain/Value/NextRelease.php

```diff
@@ -160,6 +160,14 @@ public function isNeeded(): bool
         return $this->nextTag->toString() !== $this->currentTag->toString();
     }
 
+    public function canBeReleased(): bool
```

Comment:
A PR like https://github.com/sonata-project/SonataAdminBundle/pull/6397 is needed before a release
- Updating the changelog
- Changing the 3.x occurences

The two solutions is see would be
- Make this PR automatic (but how ?)
- Using a label for the pre-release PR

Do you have any other idea ?